BiweeklyMeetings/2022-02-15

```diff
@@ -0,0 +1,334 @@
+**Table of Contents:** 
+
+- [Summary](#summary)
+- [Editors Meeting Flow](#editors-meeting-flow)
+- [February 15 2022 notes](#february-15-2022-notes)
+  * [Triage](#triage)
+    + [PR136 - metadata onchain standard ](#pr136)
+    + [PR153 - moving CIP 23 status to 'Proposed'](#pr153)
+    + [PR157](#pr157)
+  * [Last Check](#last-check)
+    + [PR148 - Update to CIP 30 (signData API)](#pr148)
+     
+  * [Review](#review)    
+    + [PR159 - tentative CIP 31: "Reference Inputs"](#pr159)  
+    + [PR160 - tentative CIP 32: "Inline Datums"](#pr160)
+    + [PR163 - Tentative CIP: "Dynamic Saturation based on Pledge"](#pr163) 
+  * [Discussions](#discussions)
+    + [Pr151 - CIP30 event API](#pr151)
+  * [Close](#close)
+- [Extra](#extra)
+  * [Current CIPs in the CIP repository and their status](#current-cips-in-the-cip-repository-and-their-status)
+  * [CIP creation process as a Sequence Diagram](#cip-creation-process-as-a-sequence-diagram)
+  * [Understanding CIPs further](#understanding-cips-further)
+
+## Summary
+
+Rough writeup of 02/15/22 Editors meeting notes taken during that day's CIP meeting, to increase transparency and dialogue with the community regarding proposed changes, implementations and considerations.  
+<sub>_Notes might contain errors or miss pieces - call out issues as needed_
+</sub>  
+Editors meetings are [public](https://www.crowdcast.io/cips-biweekly), [recorded](https://www.crowdcast.io/e/cip-editors-meeting-39) and [summarized](https://github.com/cardano-foundation/CIPs/tree/master/BiweeklyMeetings): do join and participate for discussions/PRs of significances to you.  
+
+
+## Editors Meeting Flow
+
+- [x] **Triage/Review**: Some CIPs might fall out of grace or not get updated, a CIP that hasn’t seen activity for 3 months should be checked on, and appropriate action taken. Ex: did any of the recent changes obsolete current CIPs? Consider ‘Active’ -> ‘Obsolete’ transitions..  
+- [x] **Last Check**: Review of the PRIOR meetings Decisions  - if no objection, apply change (effectively a two week lag from decision to action, as a grace period)  
+- [x] **New CIPs Review**: CIPs up for review should be looked over collectively, with discussion where needed. (on top of the asynchronous reviews)  
+PR -> ‘Draft’: Needs format + approval.  
+‘Draft’ -> ‘Proposed’: Needs a PLAN towards Active + implementation.  
+‘Proposed’ -> ‘Active’:  Objective criteria as laid out observed, and consensus agreeing.   
+- [x] **Current Discussions**: What the current CIPs discussions are on social media / forums / Discord.  
+- [x] **Close**: Recap of actions taken and decisions. List the CIPs that are due for review.  Distribution of the minutes via mailing list.  
+
+## February 15 2022 notes 
+
+**Attending Editors**: Matthias Benkort,  Sebastien Guillemot, Frederic Johnson, Maria Jover + guests (MPJ)
```

Comment:
Maria - not an Editor. The title is significant. 
Please list the missing editors in notes (Robert and Duncan).